### PR TITLE
Construct alternate login path

### PR DESCRIPTION
**Pull request by nbarnabee**
Hey, @HWaruguru I figured it was easier just to show you what I tried rather than try to explain it.

Right now the button will change on login, though there's a flash of the old button, which is kind of annoying.  I'm not using localStorage at all and we'd need to pass the value of currentUser to other components somehow and I'm not sure how easy or difficult that would be.

### CODE REVIEW

**Comment by nbarnabee** [comment]
> we'd need to pass the value of currentUser to other components somehow and I'm not sure how easy or difficult that would be.

Thinking more on this, a better way might be to attach the `onMount -> call getLoggedInUser()` to the main app, and then it could pass the value of currentUser down to all its children.  I don't have time to mess around with that right now (and I don't know if it would even be a good idea); it's just a thought.

File: src/stores/api.js [hunk 1]
```diff
@@ -31,35 +33,16 @@ export async function getAllTimeGreat(lastThirtyDays=false) {
...
-  if (!localStorage.getItem("currentUser")) {
-    window.open("http://localhost:8082/api/login");
-  }
-
-  location.reload();
-}
-
 export function LogOut() {
+  window.location.href = "http://localhost:8082/api/logout";
   localStorage.removeItem("currentUser");
```
Comment (HWaruguru) [inline src/stores/api.js:47]: You should also remove this localStorage

**Comment by HWaruguru** [comment]
I have compared these two PRs and this @nbarnabee's looks better. We can work with this.

File: src/stores/api.js (hunk 1 above)
Comment (nbarnabee) [reply src/stores/api.js:47]: Oh yeah.  🤣

**Comment by nbarnabee** [comment]
This is based off of your branch in #33, @HWaruguru so we can go ahead and merge it if you're happy with that.

**Pull request merged**